pkgs/test/lib/src/runner/browser/chrome.dart

```diff
@@ -73,6 +82,8 @@ class Chrome extends Browser {
         if (port != null) {
           remoteDebuggerCompleter.complete(
               getRemoteDebuggerUrl(Uri.parse('http://localhost:$port')));
+
+          connectionCompleter.complete(await _connect(process, port, idToUrl));
```

Comment:
I don't think this `await` is necessary, `Completer.complete` takes a `FutureOr`.

Reply:
Done.